### DISCUSSION TITLE
Local installation not working

### DISCUSSION THREAD

**Discussion by aalainn** [statamic/cms#4563] [Q&A] [answered]
Hi
I followed the instructions of installing Statamic.
I'm using Visual Studio Code as a Codeeditor.

I named my project "test-1".
There is PHP 7.4.9  on my machine.
OS Win10.
When entering test-1.test (the video tutorial does not explain why to add ".test" hence I have no idea what I'm doing) the Browser throws an error: Site cannot be reached (DNS_PROBE_FINISHED_NXDOMAIN).

Does anybody have an idea what I do wrong?

Kind regards
Alain

**Comment by duncanmcclean** [comment, marked as answer]
Do you have a web server setup?

**Reply by aalainn** [reply]
You mean something like XAMPP?

**Reply by duncanmcclean** [reply]
Yeah (although not sure how well XAMPP will work with Statamic). 

You can try running `php artisan serve`, it will give you a webserver. It'll give you a local server at http://127.0.0.1:8000

**Reply by aalainn** [reply]
Ok, I will try that.
But how does it work localy in general?
Without XAMPP?
Without starting a server manually?

**Reply by duncanmcclean** [reply]
I'm not sure what the recommended ways are on Windows. However, on mac I use [Laravel Valet](https://github.com/laravel/valet). I believe there's a fork for Windows.

**Reply by jasonvarga** [reply]
https://laragon.org is a good one for Windows.

**Reply by ronvdc** [reply]
I'm using Laragon on Windows 11, and it works great with local Statamic installations.
I particularly like the fact that you can conveniently switch web roots, so you can work with different sites very easily.

**Reply by aalainn** [reply]
Aha, but isn't this something that should be mention in the installation-documentation of statamic?
Edit:
I just saw that is under "Requirements". In my opinion this should also been mention under "Installation" for dumbheads like me ;-)
